src/databricks/labs/ucx/hive_metastore/hms_lineage.py

```diff
@@ -0,0 +1,39 @@
+import base64
+import time
+
+from databricks.sdk import WorkspaceClient
+
+
+class HiveMetastoreLineageEnabler:
+    def __init__(self, ws: WorkspaceClient):
+        self._ws = ws
+
+    def _get_init_script_content(self):
+        _init_script_content = """if [[ $DB_IS_DRIVER = "TRUE" ]]; then
+  driver_conf=${DB_HOME}/driver/conf/spark-branch.conf
+  if [ ! -e $driver_conf ] ; then
+    touch $driver_conf
+  fi
+cat << EOF >>  $driver_conf
+  [driver] {
+   "spark.databricks.dataLineage.enabled" = true
+   }
+EOF
+fi"""
+        return base64.b64encode(_init_script_content.encode()).decode()
+
+    def _add_global_init_script(self):
+        self._ws.global_init_scripts.create(
```

Comment:
Do we check for existing init scripts? Can we append to existing scripts? Is there a risk?

Reply:
+1, it's an easy add

Reply:
Hi @FastLee and @nfx - are you referring to adding the portion in an existing **Global** init scripts?

Reply:
Keep scripts separate, yes. But don't add lineage enabler if it's already there. Manual reproduction: run installer twice and verify only one script created by ucx

Reply:
Hi @nfx - This is what the code does -
if spark config present and the init script is enabled - skips creating a new one
If spark config present but the init script is disabled - asks if the user wants to enable it and enables/leave as it is based on user response
If spark config is not present then creates a new enabled global init script.